pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/api/TemplateResource.kt

```diff
@@ -27,7 +27,17 @@ class TemplateResource<Kode : Enum<Kode>, out T : BrevTemplate<BrevbakerBrevdata
     templates: Set<T>,
     private val laTeXCompilerService: LaTeXCompilerService,
 ) {
-    val templates: Map<Kode, T> = templates.associateBy { it.kode }
+    private val templates: Map<Kode, T> = templates.associateBy { it.kode }
+
+    fun listTemplatesWithMetadata() = templates.map { it.value.description() }
```

Comment:
Det var dette jeg tenkte at kanskje burde respektere hva feature-togglen velger av template versjon, siden en ny versjon kan ha endringer i metadata.
`templates.keys.map { getTemplate(it).description() }`

Reply:
Ah, sånn ja. Det er jo eit godt poeng, oppdaterer.